__tests__/get-articles-api.test.js

```diff
@@ -57,4 +57,20 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("404: responds with an error msg when topic doesn't exist", () => {
+    return request(app)
+      .get("/api/articles?topic=chaitali")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+  test("404: responds with an error msg when there are no article for the given topic", () => {
+    return request(app)
+      .get("/api/articles?topic=paper")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });

```

Comment:
It would be more appropriate for your endpoint to return an empty array of articles with a 200 status code than to throw an error